pkg/client/retrieve.go

```diff
@@ -141,8 +189,9 @@ func UntarAll(reader io.Reader, destFile, prefix string) (filenames []string, re
 		}
 		entrySeq++
 		mode := header.FileInfo().Mode()
-		outFileName := path.Join(destFile, header.Name[len(prefix):])
+		outFileName := filepath.Join(destFile, header.Name)
```

Comment:
Double check prefix

Reply:
So this _kind of_ doesnt really matter. The reason being that we take the base name right below this so it is a bit redundant to also strip the prefix.

I'll undo this change for now, but not sure how this may change as both OSs are supported.